src/tests/utils/ApiConstants.ts

```diff
@@ -5,4 +5,8 @@ export const constants = {
 	DEV_BAV_MISSING_SUB_ACCESS_TOKEN: process.env.DEV_BAV_MISSING_SUB_ACCESS_TOKEN,
 	DEV_BAV_TEST_HARNESS_URL: process.env.DEV_BAV_TEST_HARNESS_URL,
 	DEV_BAV_SESSION_TABLE_NAME: "session-bav-cri-ddb",
+	DEV_IPV_BAV_EXPIRED_TOKEN: process.env.DEV_IPV_BAV_EXPIRED_TOKEN,
+	DEV_IPV_BAV_MISSING_SUBJECT_TOKEN: process.env.DEV_IPV_BAV_MISSING_SUBJECT_TOKEN,
+	DEV_IPV_BAV_MISSING_CLIENT_ID_TOKEN: process.env.DEV_IPV_BAV_MISSING_CLIENT_ID_TOKEN
+
```

Comment:
I believe these will need to be added to the `run-tests.sh` script so that the tests pass in the container?